.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Executables
+on:
+  release:
+    types: [published]
+jobs:
+  lint:
+    name: Build Executables
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install gon via HomeBrew for code signing and app notarization
+        run: |
+          brew tap mitchellh/gon
+          brew install mitchellh/gon/gon
+      - run: ./scripts/build.sh
+        env:
+          APPLE_DEV_CERT: ${{secrets.APPLE_DEV_CERT}}
+          APPLE_ID_USERNAME: ${{secrets.APPLE_ID_USERNAME}}
+          APPLE_ID_KEY: ${{secrets.APPLE_ID_KEY}}
+      - uses: actions/upload-artifact@v3
+        with:
+          name: macos executable
+          path: percy-macos
```

Comment:
zip file can be

`percy-darwin.zip`
`percy-linux.zip`
`percy-win.zip`

all should unzip to `percy` executable (percy.exe for win)

Reply:
updated

Reply:
`percy` isn't executable currently it seems, does the binary need `chmod +x` while building?

Reply:
updated. we need to convert to tar format to retain permissions.